### ISSUE TITLE
Use version IRI in rdfs:isDefinedBy

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#383]
The use of rdfs:isDefinedBy was proposed for two reasons, at least as I remember: 

1. Before de-modularization, to indicate which ontology module defines a given term.
2. To version terms in the absence of a versioned IRI for each term.

Currently we are adding the unversioned IRI. I believe ontology-toolkit already provides an option to use the versioned IRI.  In that case, this simply needs a change to bundle.yaml to configure it.

**Comment by sa-bpelakh** [comment]
That is correct, just add `versionedDefinedBy: true` to the `definedBy` action.

**Comment by rjyounes** [comment]
It turns out there's nothing to do to the bundle config file, since this is a commandline argument. The task is merely to update the build instructions in Confluence.

**Comment by rjyounes** [comment]
Looks like `--versioned-definedBy` is not an option for the `bundle` command. @sa-bpelakh How can this be invoked during bundling?
